app/jest/my-dashboard/my-projects/roleManagement.test.tsx

```diff
@@ -0,0 +1,46 @@
+import { render, screen, fireEvent, waitFor } from '@testing-library/react';
+import RoleManagement from 'page-partials/my-dashboard/RoleManagement';
+import { sampleRequest } from '../../samples/integrations';
+import { listClientRoles, KeycloakUser } from 'services/keycloak';
+
+const mockClientRolesResult = ({ email, username, attributes }: KeycloakUser) => {
+  return [(email = 'mockEmail@gov.bc.ca'), (username = 'mockUserName'), (attributes = null)];
+};
+
+jest.mock('services/keycloak', () => ({
+  listClientRoles: jest.fn(() => Promise.resolve([mockClientRolesResult, null])),
+}));
+
+describe('role management tab', () => {
+  it('should match the display data', () => {
```

Comment:
@kurokn please write accurate descriptions as not sure what `display data` means?